options.go

```diff
@@ -0,0 +1,32 @@
+package gosundheit
+
+type Option func(*health)
+
+// WithCheckListener allows you to listen to check start/end events
+func WithCheckListener(listener CheckListener) Option {
+	return func(h *health) {
+		if listener != nil {
+			h.checksListener = listener
+		}
+	}
+}
+
+func withDefaultCheckListener() Option {
```

Comment:
We already initialize the struct with the default listener in `New()`

Reply:
Will remove the step in `New()` as it's better that all defaults are in the same location